app/views/archived/petitions/show.html.erb

```diff
@@ -72,6 +72,73 @@
     </div>
 
     <div class="about-petitions">
+      <% if debate_outcome = @petition.debate_outcome %>
```

Comment:
I haven't done a full like-for-like check but parts of this section appear similar to the parts of the view for non-archived petitions such as `_debate_threshold.html.erb` - is there enough commonality for us to reuse one or more of the partials here?